src/components/WonderCart.js

```diff
@@ -94,7 +94,7 @@ export function WonderCart(props) {
         cardLink={
           (canAccessGlobalCTB &&
             !hasYithExtended &&
-            NewfoldRuntime.sdk.ecommerce.brand_settings.wondercartBuyNow) || ''
+            NewfoldRuntime.ecommerce.brand_settings.wondercartBuyNow) || ''
```

Comment:
In NewfoldRuntime, we're not getting ecommerce value currently, can you make those changes as well ?

Reply:
done